### PR TITLE
Created system-upgrade-controller image and script.

### PR DESCRIPTION
**Pull request by phillipsj**
The script supports traditional way that wins can upgrade itself and
host process containers.

Signed-off-by: Jamie Phillips <jamie.phillips@suse.com>

### CODE REVIEW

File: package/windows/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,9 @@ SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $P
 RUN New-Item -ItemType SymbolicLink -Target "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -Path "C:\Windows\System32\WindowsPowerShell\v1.0\pwsh.exe"
 
 COPY ./wins.exe /Windows/
+COPY ./wins.exe wins.exe
+COPY ./nstall.ps1 install.ps1
```
Comment (rosskirkpat) [inline package/windows/Dockerfile:13]: typo with `nstall.ps1`

**Comment by phillipsj** [comment]
[Linking GH #181](https://github.com/rancher/windows/issues/181)

**Review by rosskirkpat — CHANGES_REQUESTED** [review]
LGTM other than typo :)

**Comment by phillipsj** [comment]
image

**Review by rosskirkpat — APPROVED** [review]
LGTM!

**Pull request merged**